bokehjs/src/coffee/renderer/tile/image_pool.coffee

```diff
@@ -0,0 +1,23 @@
+class ImagePool
```

Comment:
great catch. I have removed the class from `tile_utils` and will also be removing `tile_utils` all together once I refactor out the `ProjectionUtils` class in favor of Proj4